### PR TITLE
drivers: usb_dc_nrfx: cleanup logging and fix style

### PR DESCRIPTION
**Pull request by jfischer-no**
Cleanup logging and fix style. Use hex format specifier
for endpoint address. Remove periods from log messages.

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

**Review by nashif — APPROVED** [review]

File: drivers/usb/device/usb_dc_nrfx.c [hunk 1]
```diff
@@ -1312,8 +1313,7 @@ int usb_dc_attach(void)
 	err = nrfx_usbd_init(usbd_event_handler);
 
 	if (err != NRFX_SUCCESS) {
-		LOG_DBG("nRF USBD driver init failed. Code: %d.",
-			(u32_t)err);
+		LOG_DBG("nRF USBD driver init failed. Code: %d", (u32_t)err);
```
Comment (finikorg) [inline drivers/usb/device/usb_dc_nrfx.c]: btw, why do you make type cast?

File: drivers/usb/device/usb_dc_nrfx.c [hunk 2]
```diff
@@ -665,7 +665,7 @@ static int eps_ctx_init(void)
 			err = k_mem_pool_alloc(&ep_buf_pool, &ep_ctx->buf.block,
 					       MAX_EP_BUF_SZ, K_NO_WAIT);
 			if (err < 0) {
-				LOG_ERR("EP buffer alloc failed for EPIN%d", i);
+				LOG_ERR("Buffer alloc failed for EP 0x%02x", i);
```
Comment (finikorg) [inline drivers/usb/device/usb_dc_nrfx.c]: is this endpoint index so we do not know direction?

File: drivers/usb/device/usb_dc_nrfx.c (hunk 1 above)
Comment (jfischer-phytec-iot) [reply drivers/usb/device/usb_dc_nrfx.c]: err is from type nrfx_err_t

File: drivers/usb/device/usb_dc_nrfx.c (hunk 2 above)
Comment (jfischer-phytec-iot) [reply drivers/usb/device/usb_dc_nrfx.c]: :scream: I will fix it in the next cleanup PR.

**Pull request merged**